README.md

```diff
@@ -2,7 +2,20 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ## Getting Started
 
-First, run the development server:
+Clone the repository.
+
+Install NPM.
+
+On Ubuntu 22.04, the default installed version of NodeJS is too old.
+
```

Comment:
### Setup JS env

If you're new to JavaScript environment, you might want to install [nvm](https://github.com/nvm-sh/nvm) which is a version manager for Node.js

You can follow the installation guide here: https://github.com/nvm-sh/nvm#installing-and-updating

After that, you will be able to run `nvm install <node version>` and `nvm use <node version>` to install and switch between node versions.

### Install the package manager

This project use `yarn` to manage library dependencies with 

```basj
npm install --global yarn
```

### Run the project

The following commend will install all the dependencies and run the project in `localhost:3000`
```bash
yarn dev
```